### ISSUE TITLE
Devspace UI Terminal Column Width

### ISSUE THREAD

**Issue by lots0logs** [devspace-cloud/devspace#1190]
**What happened?**  
No matter how large your window is the content width inside the terminal seems to be limited to ~80 characters.

**What did you expect to happen instead?**  
The content width would use the available size of the terminal div.

**How can we reproduce the bug?** (as minimally and precisely as possible)  
No special steps needed. Simply open a terminal and start typing until it cuts off at ~80 characters. 

**Local Environment:**  
- DevSpace Version: 5.1.0
- Operating System: linux
- Deployment method: helm

**Kubernetes Cluster:**  
- Cloud Provider: other
- Kubernetes Version: 
   ```
   Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.0", GitCommit:"e19964183377d0ec2052d1f1fa930c4d7575bd50", GitTreeState:"clean", BuildDate:"2020-09-30T19:31:27Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
   Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.6", GitCommit:"dff82dc0de47299ab66c83c626e08b245ab19037", GitTreeState:"clean", BuildDate:"2020-07-15T16:51:04Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
   ```

**Anything else we need to know?**  
![2020-10-07_15-47](https://user-images.githubusercontent.com/4675662/95386599-29e66580-08b5-11eb-880b-c91f75d3803a.png)

/kind bug

**Comment by FabianKramm** [comment]
@lots0logs thanks for reporting this issue! The problem is that the terminal that is opened by the server in the background stays the same size. To solve this we would need an additional server endpoint that resizes the backend terminal to the size of the browser terminal. I'll add this to the todo list

**Comment by lots0logs** [comment]
Thank you! One other thing I forgot to mention is that if you output the contents of a file using `cat` command it does use the entire terminal width but only for that output.

**Comment by FabianKramm** [comment]
Fixed with v5.1.2
